### ISSUE TITLE
nw.App.on("open",function(args) not firing on windows 10 , sdk 0.17.4

### ISSUE THREAD

**Issue by razzbee** [nwjs/nw.js#5354]
The onOpen event is not working for me on windows 10 sdk 0.17.4 , the docs says the app needs to be packed but still it doesnt work after packaging it :

nw.App.on("open",function(args){
  console.log("On Opened Triggered");
});

**Comment by ghostoy** [comment]
@razzbee It works for me.

`open` event is triggered only when you open a second instance of your app. For example, first start your app and you will **NOT** receive this event. Then start your app again without closing first one and you will now receive this event in your first app.

**Comment by razzbee** [comment]
Yes , that was exactly what I did , but seems nothing is happening

**Comment by ghostoy** [comment]
Could you post a full sample app you are testing with?

**Comment by razzbee** [comment]
ok

**Comment by razzbee** [comment]
I have been able to fix it , though it is very strange , event wont trigger if : 
```json
single-instance: false 
```
in the manifest (package.json) file ,so single-instance:true is needed for the event to fire

**Comment by ghostoy** [comment]
@razzbee `single-instance` should not have effect. If so, we will fix it.
